src/app/static/src/js/components/reports/reportDependencies.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div v-if="childDependencies.length">
+        <hr/>
+        <div class="row">
+            <h4>Dependencies</h4>
+            <div class="col-12">
+                <report-dependency-list :dependency-list="childDependencies"></report-dependency-list>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+    import {api} from "../../utils/api";
+    import {ReportDependencies, ReportDependency} from "../../types";
+    import Vue from "vue";
+    import ReportDependencyList from "./reportDependencyList.vue";
+
+    interface Data {
+        dependencies: ReportDependencies | null
+    }
+
+    interface Computed {
+        childDependencies: ReportDependency[]
+    }
+
+    export default Vue.extend<Data,{}, Computed, {}>({
+        name: "reportDependencies",
+        props: ['report'],
+        data: () => {
+            return {
+                dependencies: null
+            }
+        },
+        computed: {
+          childDependencies(){
```

Comment:
The top level 'dependency_tree' value always has report in question as the root item, but we don't want to display this report as its own dependency so we start from the next level down

Reply:
If it's worth a comment in the PR then for the sake of posterity it might be worth a comment in the code itself